docs/reference/edot-cloud-forwarder/gcp.md

```diff
@@ -0,0 +1,285 @@
+- `secretmanager.versions.destroy` 
+- `secretmanager.versions.enable` 
+- `secretmanager.versions.get` 
+:::
+
+:::{dropdown} Pub/Sub
+These permissions are needed for Pub/Sub management: 
+- `pubsub.subscriptions.create` 
+- `pubsub.subscriptions.delete` 
+- `pubsub.subscriptions.get` 
+- `pubsub.subscriptions.getIamPolicy` 
+- `pubsub.subscriptions.list` 
+- `pubsub.subscriptions.setIamPolicy` 
+- `pubsub.subscriptions.update` 
+- `pubsub.topics.attachSubscription` 
+- `pubsub.topics.create` 
+- `pubsub.topics.delete` 
+- `pubsub.topics.detachSubscription` 
+- `pubsub.topics.get` 
+- `pubsub.topics.getIamPolicy` 
+- `pubsub.topics.setIamPolicy` 
+- `pubsub.topics.update` 
+:::
+
+:::{dropdown} Cloud Run
+The following permissions are needed for Cloud Run management: 
+- `run.operations.get` 
+- `run.services.create` 
+- `run.services.delete` 
+- `run.services.get` 
+- `run.services.getIamPolicy` 
+- `run.services.setIamPolicy` 
+- `run.services.update` 
+:::
+
+:::{dropdown} Service Account
+The following permissions are needed for Service Account management: 
+- `iam.serviceAccountKeys.create` 
+- `iam.serviceAccountKeys.get` 
+- `iam.serviceAccounts.create` 
+- `iam.serviceAccounts.delete` 
+- `iam.serviceAccounts.get` 
+- `iam.serviceAccounts.update` 
+- `iam.serviceAccounts.actAs` 
+:::
+
+:::{dropdown} Artifact Registry
+The following permissions are needed: 
+- `artifactregistry.repositories.create` 
+- `artifactregistry.repositories.delete` 
+- `artifactregistry.repositories.get` 
+- `artifactregistry.repositories.getIamPolicy` 
+- `artifactregistry.repositories.setIamPolicy` 
+- `artifactregistry.repositories.update` 
+- `artifactregistry.repositories.downloadArtifacts` 
+:::
+
+
+## Quick start
+
+Follow these steps to get started with {{edot-cf}} for GCP.
+
+:::::{stepper}
+
+::::{step} Authenticate with GCP
+
+You need to be authenticated on the Google project in which you want to deploy {{edot-cf}} for GCP. If you have `gcloud` installed already, run this command:
+
+```shell
+gcloud auth application-default login
+```
+
+Otherwise, follow [Google instructions](https://docs.cloud.google.com/sdk/docs/install-sdk) for the `gcloud` use and configuration.
+::::
+
+::::{step} Configure Terraform
+Deploy {{edot-cf}} for GCP using the [Terraform module]((https://github.com/elastic/terraform-google-edot-cloud-forwarder). 
+
+First create a providers file, `providers.tf`, then configure the [`google` provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/getting_started/guides/getting_started):
+
+```ini subs=true
+provider "google" {
+  project = "[GCP project]"
+  region  = "[GCP region]"
+}
+```
+
+This ensures that you can deploy the Google resources on your project. Then create a new Terraform file, `main.tf`, and add the {{edot-cf}} for GCP module:
+
+```ini subs=true
+module "ecf" {
+  source = "github.com/elastic/terraform-google-edot-cloud-forwarder?ref=v{{version.edot_cf_gcp}}"
```

Comment:
@theletterf Do you see this `version.edot_cf_gcp`? I want this to point to the latest version of ECF GCP terraform, so `latest-terraform-google-edot-cf-version` from this [PR](https://github.com/elastic/docs-builder/pull/2342/files) of yours. So should I change `version.edot_cf_gcp` to something else?